.github/workflows/push-pr_workflow.yml

```diff
@@ -0,0 +1,49 @@
+name: Python CI
```

Comment:
This looks like it has the right components, but I don't see that GitHub actions is actually being triggered on this PR. How can we verify that this is right? Do we need to enable actions in this repo?

Reply:
We may have to disable the travis server to get this to work

Reply:
Wouldn't this first need to be merged to `develop` or `main` to take over the CI?

Reply:
Yes, I discussed this on the teams chat (guessing you didn't see it with your work machien in the shop) - it does need to be merged first. We'll do that and check the pipeline after merging, should be fine since [the fork ran the pipeline just fine and had no problems](https://github.com/AlexanderWinterLLNL/merlin/actions/runs/919970896)